jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/mapper/ProcessInstanceCustomQueryMapper.java

```diff
@@ -59,7 +59,9 @@ public static ProcessInstanceCustomQueryMapper get() {
                     String varName = getColumnStringValue(dataSetResult, COLUMN_VAR_NAME, i);
                     String varValue = getColumnStringValue(dataSetResult, COLUMN_VAR_VALUE, i);
                     
-                    ((org.jbpm.kie.services.impl.model.ProcessInstanceCustomDesc)pi).addVariable(varName, varValue);
+                    if (varName != null) {
+                        ((org.jbpm.kie.services.impl.model.ProcessInstanceCustomDesc)pi).addVariable(varName, varValue);
```

Comment:
Maybe we could apply kiegroup codestyle for better readibility:
```java
((org.jbpm.kie.services.impl.model.ProcessInstanceCustomDesc) pi).addVariable(varName, varValue);
```

Reply:
Done